finalProject.html

```diff
@@ -12,22 +12,47 @@
 <body>
     <div class="leftContainer">
         <h1>Light Weather</h1>
+<<<<<<< HEAD
```

Comment:
looks like you pulled master into this branch and got conflicts which was not resolved.
You need to find now all places with this arrows and identify which is the proper code you want to keep and delete not needed.